cmd/crio/config.go

```diff
@@ -107,8 +113,17 @@ runtime_untrusted_workload = "{{ .RuntimeUntrustedWorkload }}"
 #     privileged containers are by definition trusted and will always use the
 #     trusted container runtime. If default_container_trust is set to "trusted",
 #     CRI-O will use the trusted container runtime for all containers.
+#
+# DEPRECATED: The runtime handler should provide a key to the map of runtimes,
+#     avoiding the need to rely on the level of trust of the workload to choose
+#     an appropriate runtime.
```

Comment:
let's clarify that we'll support this for 1.12, no-op maybe in 1.13 and remove it altogether in 1.14

Reply:
Yes good point, let me add this comment.

Reply:
so, I also think we should print a warning in validateConfig if this is used from now on, basically just saying what's written here so admins and users are aware of this

Reply:
Both comments addressed!